proxies/live/apiproxy/targets/ers-target.xml

```diff
@@ -163,21 +184,16 @@
         </Flow>
     </Flows>
     <HTTPTargetConnection>
-      <SSLInfo>
-{% if '--ft-' in (ERS_TARGET_SERVER | default('e-referrals-service-api')) %}
-        <TrustStore>ref://e-referral-service-ca</TrustStore>
-{% endif %}
-        <Enabled>true</Enabled>
-      </SSLInfo>
-    <LoadBalancer>
-      <Server name="{{ ERS_TARGET_SERVER | default('e-referrals-service-api') }}"/>
-    </LoadBalancer>
-    <Path>/ers-api</Path>
-    <Properties>
-      <Property name="supports.http10">true</Property>
-      <Property name="request.retain.headers">User-Agent,Referer,Accept-Language</Property>
-      <Property name="retain.queryparams">apikey</Property>
-    </Properties>
+        <SSLInfo>{% if '--ft-' in (ERS_TARGET_SERVER | default('e-referrals-service-api')) %} <TrustStore>ref://e-referral-service-ca</TrustStore> {% endif %} <Enabled>true</Enabled></SSLInfo>
```

Comment:
I assume this is just formatting;

Reply:
I think this was the xml formatter. I'll try turning it off and leaving the jinja templates as before